### ISSUE TITLE
Allow to select webspace security system when editing role

### ISSUE THREAD

**Issue by nnatter** [sulu/sulu#5161]
| Q | A
| --- | ---
| Sulu Version | 2.0.6

# Actual Behavior

Sulu allows to configure a `Security System` inside of a [webspace configuration](https://github.com/sulu/sulu/blob/master/src/Sulu/Component/Webspace/Loader/schema/webspace/webspace-1.1.xsd#L16).
Although i can configure such a system, I cannot select it in the `System` dropdown when creating/editing a `User Role` via the Sulu admin interface.

# Expected Behavior

I would expect to be able to select the configured role in the `System` dropdown.

# Steps to Reproduce

1. Configure a new `Security System` inside of a webspace configuration with
```xml 
    <security>
        <system>Webspace System</system>
    </security>
```

2. Create a new Role via `Settings / User Roles / Add`

**Comment by alexander-schranz** [comment]
duplicated to #4719 will close the other issue.

My Proposel would the `PageAdmin` should create the missing Security Contexts like the Community Bundle does it currently.

**Comment by danrot** [comment]
Fixed in #5171
